FirebaseAdmin/FirebaseAdmin.Tests/Util/AuthUtilTest.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright 2021, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using Xunit;
+using static FirebaseAdmin.Auth.Utils;
+
+namespace FirebaseAdmin.Auth.Tests
+{
+    public class AuthUtilTest : IDisposable
```

Comment:
Just `UtilsTest` to mirror the name of the impl class. This is already scoped to `FirebaseAdmin.Auth.Tests`, so there's no ambiguity.

Reply:
Done